corehq/apps/app_manager/templates/app_manager/partials/add_new_module_modal.html

```diff
@@ -27,103 +27,25 @@ <h4 class="modal-title" id="modal-title">
         </h4>
       </div>
       <div class="modal-body modal-flex-container">
-        <button type="button" class="new-module-option" data-type="survey">
-          <div class="panel panel-default">
-            <div class="panel-heading">{% trans "Surveys" %}</div>
-            <div class="panel-body">
-              <i class="fa-regular fa-file"></i>
-              <p>{% blocktrans %}Collect data once{% endblocktrans %}</p>
-            </div>
-          </div>
-        </button>
-        <button type="button" class="new-module-option" data-type="case">
-          <div class="panel panel-default">
-            <div class="panel-heading">{% trans "Case List" %}</div>
-            <div class="panel-body">
-              <i class="fa-solid fa-list-check"></i>
-              <p>{% blocktrans %}Track items over time{% endblocktrans %}</p>
-            </div>
-          </div>
-        </button>
+        {% include 'app_manager/partials/module_option_button.html' with module_type="survey" heading="Surveys" icon_class="fa-regular fa-file" description="Collect data once" %}
```

Comment:
do we translate Surveys and description now? it seems like translation is missed now? It might be better create a base template for the option and extend the base template to add the text. It's important to have the text translated.

Reply:
Another option is to see if passing in `module_type=_("Survey")` works.

Reply:
Thanks Jenny, it doesn't work ><... it will render the `{% include ...%}` literally. I added base template in 473d914